tests/qa_automation/kernel_multipath.pm

```diff
@@ -15,11 +15,14 @@ use strict;
 use testapi;
 use utils;
 use iscsi;
+use registration 'add_suseconnect_product';
+use version_utils "is_sle";
 
 sub start_testrun {
     my $self = shift;
 
-    zypper_call("in open-iscsi qa_test_multipath");
+    add_suseconnect_product('sle-module-python2') if (is_sle(">15"));
```

Comment:
really ? this isn't 'SLE-Module-Legacy"

Reply:
Really

Reply:
http://10.100.12.105/tests/1586#step/kernel_multipath/25
Activating sle-module-python2 15.1 x86_64 ...
-> Adding service to system ...
-> Installing release package ...
Successfully registered system

Reply:
```
s390vsw073:~ # zypper search-packages python-xml
Following packages were found in following modules:

Package         Module or Repository                                   SUSEConnect Activation Command                        
--------------  -----------------------------------------------------  ------------------------------------------------------
python-xml      Basesystem Module (sle-module-basesystem/15.1/s390x)   SUSEConnect --product sle-module-basesystem/15.1/s390x
python-xmldiff  SUSE Package Hub (PackageHub/15.1/s390x)               SUSEConnect --product PackageHub/15.1/s390x           
python-xml      Available in repo sle-module-basesystem:15.1::product                                                        

To activate the respective module or product, use SUSEConnect --product.
Use SUSEConnect --help for more details.
```

Reply:
Interesting incosistency, i'm adding WIP label until it is sorted out.

Reply:
It seems like your s390x installation is out of date. https://build.suse.de/package/view_file/SUSE:SLE-15-SP1:GA/000product/sle_module_python2.group?expand=1 shows that python-xml is part of the python2 module and also the .kiwi and .kwd files next to that file show that.